pkg/azurefile/azurefile.go

```diff
@@ -268,3 +268,14 @@ func checkShareNameBeginAndEnd(fileShareName string) bool {
 
 	return false
 }
+
+// get snapshot name according to snapshot id, e.g.
+// input: "rg#f5713de20cde511e8ba4900#csivolumename#2019-08-22T07:17:53.0000000Z"
+// output: 2019-08-22T07:17:53.0000000Z
+func getSnapshot(id string) (string, error) {
+	segments := strings.Split(id, seperator)
+	if len(segments) < 4 {
```

Comment:
I think we don't allow one more `#` :
`if len(segments) != 4 {`

Reply:
Ok. The unit test has been added in `azurefile_test.go`

Reply:
done.